core/app/models/spree/return_authorization.rb

```diff
@@ -20,6 +21,8 @@ class ReturnAuthorization < Spree::Base
     validate :must_have_shipped_units, on: :create
     validate :no_previously_exchanged_inventory_units, on: :create
 
+    validates :recipient, presence: true, if: :override_recipient
+    attr_accessor :override_recipient
```

Comment:
Here's what it looks like without the override checked.

![screenshot_11_25_15__4_46_pm](https://cloud.githubusercontent.com/assets/24883/11410274/14d3811e-9394-11e5-8ecd-e4b082866ad0.png)